sound/soc/codecs/rt715-sdca.c

```diff
@@ -129,18 +138,23 @@ static int rt715_sdca_set_amp_gain_put(struct snd_kcontrol *kcontrol,
 	}
 
 	/* Lch*/
-	ret = regmap_write(rt715->mbq_regmap, mc->reg, gain_l_val);
-	if (ret != 0) {
-		dev_err(component->dev, "Failed to write 0x%x=0x%x\n", mc->reg,
-			gain_l_val);
-		return ret;
+	for (i = 0 ; i < loop_cnt ; i++) {
```

Comment:
no, sorry, I didn't mean *missing* spaces, I meant *superfluous* spaces. It should be
```
for (i = 0; i < loop_cnt; i++) {
```
i.e. *no* space before the 2 semicolons